src/interp/interp-inl.h

```diff
@@ -218,6 +218,17 @@ void FreeList<T>::Delete(Index index) {
   free_items_++;
 }
 
+template <typename T>
+FreeList<T>& FreeList<T>::operator=(FreeList<T>&& other) {
+  list_ = other.list_;
```

Comment:
I think you would need to free all the items you are currently holding before moving the new items into place.

Given the complexity here maybe its best to just delete the copy constructor?

@steven-johnson WDYT?

Reply:
Looking at the halide use case it looks like the copy could easily be avoided by initializing `store` in the initlizer list part of `WasmModuleContents::WasmModuleContents`  rather than using `store = wabt::interp::Store(calc_features(halide_module.target()));` in the body.   So I think deleting the operator makes sense.

Reply:
You are right about deleting the items. Ok, I will wait your decision before doing further changes. Anyway, these operations needed to be done before even if the compiler did it, so reassignment was always a costly operation.

Reply:
Please go ahead and remove the assignment operations.. I'm pretty its the right approach for these classes.

We should also delete the copy constructor too.